gateway-adapter/src/main/java/org/apache/hadoop/gateway/provider/federation/jwt/filter/SSOCookieFederationFilter.java

```diff
@@ -38,18 +38,6 @@ protected void handleValidationError(HttpServletRequest request,
     super.handleValidationError(request, response, status, error);
   }
 
-  /**
-   * Encapsulate the acquisition of the JWT token from HTTP cookies within the
-   * request.
-   *
-   * @param req servlet request to get the JWT token from
-   * @return serialized JWT token
-   */
-  @Override
-  protected String getJWTFromCookie(HttpServletRequest req) {
-    return super.getJWTFromCookie(req);
-  }
-
```

Comment:
Will this break backwards compat if someone upgrades? I don't quite understand how these adapter classes work.

Reply:
I'm assuming no since there is no reason this method gets called anymore since its deleted everywhere?

Reply:
It should not break backwards compatibility, looks good.